src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -167,6 +187,17 @@ export default {
 				})
 		},
 
+		async toggleReadStatusPrivacy() {
+			try {
+				await this.$store.commit(
```

Comment:
this should be a dispatch of that action, then the action does the commit.
I don't think this code works as it would update the value directly without going through the action

Reply:
also: would be good to disable the checkbox while saving.
then when saving completed, display a toast notification "Your privacy settings were saved" which inspires confidence.

we used the same pattern for conversation settings dialog

otherwise when I click the checkbox I'm leaving wondering whether it's saved and every time I find myself scrolling down the dialog to find the save button.

=> could be done separately

Reply:
let me adjust this, quick change

Reply:
https://github.com/nextcloud/spreed/pull/4231/commits/bd9aead8c1e1eb94e665c3ec44279757a21ebcfa

Reply:
Ah yeah it worked before as the action was basically skipped it still worked. But now the backend request is missing 🙈